src/content/account/billing.mdx

```diff
@@ -90,7 +90,7 @@ Each plan comes with a specific number of snapshots per month. If you reach that
 
 1. **Free plan:** Review and testing will be paused once you exhaust the 5,000 free snapshots per month. You can resume testing by upgrading to a premium plan or waiting until the limit resets at the end of the billing cycle. ![Builds blocked because the account hit the monthly snapshot limit](../../images/build-blocked.png)
 
-2. **Premium plans:** Review and testing will not be interrupted. Any additional snapshots beyond the included amount will be billed at the end of the month.
+2. **Paid plans:** Review and testing will not be interrupted. Any additional snapshots beyond the included amount will be billed at the end of the month.
```

Comment:
@winkerVSbecks, feel free to disregard this, but as we're renaming this from premium plans/accounts, shouldn't the "Free plan" above also reflect this change? And if that's the case, it might be worth taking a pass at the rest of the docs and seeing if there's other references (e.g., [`mandatory-pr-checks.md`](https://github.com/chromaui/chromatic-docs/blob/main/src/content/guides/mandatory-pr-checks.md))

Reply:
What would the "Free plan" change be?